src/algokit/cli/account.py

```diff
@@ -0,0 +1,161 @@
+import logging
+from pathlib import Path
+
+import click
+import keyring
+
+from algokit.core.account import (
+    KEYRING_KEY,
+    KEYRING_NAMESPACE,
+    DispenseApiAudiences,
+    get_oauth_tokens,
+    is_authenticated,
+    process_dispenser_request,
+    set_keyring_passwords,
+)
+
+logger = logging.getLogger(__name__)
+
+assets = {
+    "ALGO": {
+        "asset_id": 0,
+        "decimals": 6,
+        "description": "Algorand token on Algorand TestNet",
+    },
+}
+
+
+@click.command("logout", help="Logout account from Dispenser API access")
```

Comment:
We should double check with Alessandro he wants the top level alias, there are pros and cons

Reply:
@robdmoore as agreed, removing aliases, also renaming generic word account as a group command to `dispenser` along with command `dispense` replaced to `withdraw`